Gemfile

```diff
@@ -0,0 +1,8 @@
+source 'https://rubygems.org'
+
+gemspec
+
+group :development, :test do
```

Comment:
This is a bit old, but it's what I generally use for best practices on gemspec/bundle issues: http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/